### ISSUE TITLE
[a11y] Navigation between internal hyperlinks

### ISSUE THREAD

**Issue by HadrienGardeur** [readium/r2-navigator-swift#99]
@gregoriopellegrino commented on [Tue May 15 2018](https://github.com/readium/r2-testapp-swift/issues/106)

When click on a link in the text the focus doesn't change, so VoiceOver remains to the origin content.

Tested on iOS 11.3 with VoiceOver

---

@mickael-menu-mantano commented on [Tue May 28 2019](https://github.com/readium/r2-testapp-swift/issues/106#issuecomment-496494678)

Internal links were broken, not only with VoiceOver. It should be fixed by now in `develop`.

---

@mickael-menu-mantano commented on [Tue May 28 2019](https://github.com/readium/r2-testapp-swift/issues/106#issuecomment-496524274)

Actually I misinterpreted this issue. The link is followed properly but the VO focus stays to the previous place until the user tap on the screen.
We might need to emit the notification https://developer.apple.com/documentation/uikit/uiaccessibilityscreenchangednotification?language=objc when switching to a different location.
